tools/graph_generation/grapher.r

```diff
@@ -0,0 +1,63 @@
+install.packages("ggplot2")
+library("ggplot2")
+library("plyr")
+
+setwd("C:\\Users\\tirpi\\Downloads")
```

Comment:
wow very cryptic syntax haha, looks good though, only 2 small things: can we load this path from the command line?